modules/theme/src/themes/wso2is/apps/login-portal.overrides

```diff
@@ -165,6 +165,20 @@
             }
         }
 
+        /** Helper class to support new line */
+        .new-line-support {
+            white-space: pre-line;
+        }
+
+        /* SMS OTP interface */
+        .sms-otp-portal-layout {
```

Comment:
Let's move this to `definitions/apps/developer-portal.less`

under:

```less
.branding-preference-preview-iframe {
    .sms-otp-fragment {
     // rest
}
```

Reply:
I added this to support both the login interface and the preview section both.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/5423/commits/ffa92cc66cc0e2a9a3b0d42f9768a7c31256a003